app/controllers/solidus_stripe/webhooks_controller.rb

```diff
@@ -5,24 +5,33 @@
 
 module SolidusStripe
   class WebhooksController < Spree::BaseController
-    SIGNATURE_HEADER = "HTTP_STRIPE_SIGNATURE"
```

Comment:
I'm not against this, but is there a specific reason to remove this constant?

Reply:
To be honest, no, it's just one less thing in the public API, but happy to revert this change 👍

Reply:
If that's the concern, we can mark it as private as well. Your call.